### PR TITLE
Fix #423: switch away from LESS to CSS for all extensions that we use

### LINKED ISSUE #423: [Performance] Less LESS
screenshot 2015-08-14 11 19 22

Our startup profile is looking pretty bad.  A big part of it is Brackets rendering LESS in the browser.  We should just pre-render all LESS to CSS and/or replace it in our tree.  We don't need LESS parsing to happen in the browser.

### PR DESCRIPTION
**Pull request by humphd**
I've converted all (i.e., all we care about) the extension LESS to CSS so that we don't have to render it in the browser, which is super expensive.  The build system takes care of converting all the core LESS to CSS when we do a dist build, so this patch eliminates any loading of LESS in a dist build.

For extensions we own, I've just converted the LESS file to CSS and deleted the LESS file.  For core Brackets extensions, I've put a generated CSS file beside the LESS file, so we can still rebase and get changes.

### CODE REVIEW

File: src/extensions/default/UploadFiles/styles.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
+.drag-files-area {
```
Comment (gideonthomas) [inline src/extensions/default/UploadFiles/styles.css]: Should we instead compile the `styles.less` file to css with minification

File: src/extensions/default/UploadFiles/styles.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  width: 40%;
+  top: -5%;
+  border-top: solid 10px #dfe2e2;
+  /* match Brackets dialog background */
+  position: absolute;
+  box-sizing: border-box;
+  transform-origin: bottom;
+  animation: uploadSpinner 2s linear infinite;
+}
+@keyframes uploadSpinner {
```
Comment (gideonthomas) [inline src/extensions/default/UploadFiles/styles.css]: This won't work here right?

Comment (humphd) [inline src/extensions/default/UploadFiles/styles.css]: https://developer.mozilla.org/en/docs/Web/CSS/@keyframes, it should be fine.

File: src/extensions/default/UploadFiles/styles.css (hunk 1 above)
Comment (humphd) [inline src/extensions/default/UploadFiles/styles.css]: I can, but it means that it's basically impossible to edit/maintain.  I'm not sure we save so much by minifying that it's worth losing the ability to maintain it later.

Comment (gideonthomas) [inline src/extensions/default/UploadFiles/styles.css]: sounds good

**Comment by gideonthomas** [comment]
r+

**Pull request merged**